biggerquery/dataset_manager.py

```diff
@@ -1,10 +1,30 @@
 import uuid
 import logging
+import functools
+
 from google.cloud import bigquery
 
 from .gcp_defaults import DEFAULT_LOCATION
 
 
+class AliasNotFoundError(ValueError):
+    pass
+
+
+def handle_key_error(method):
```

Comment:
Why this decorator is not used on `collect` method?

Reply:
done